extensions/azurePublishNew/src/components/shared/autoComplete/AutoComplete.tsx

```diff
@@ -0,0 +1,818 @@
+// Copyright (c) Microsoft Corporation.
```

Comment:
did you write this? it's super general purpose. should it come from a `shared` component inside Composer?

Reply:
Eventually the Wizard & AutoComplete will be moved to the shared folder.